test/677-waterfall.py

```diff
@@ -52,4 +52,22 @@
 # Chilnualna Creek Cascades, no height
 assert_has_feature(
     14, 2748, 6344, 'pois',
-    { 'kind': 'waterfall', 'min_zoom': 14 })
\ No newline at end of file
+    { 'kind': 'waterfall', 'min_zoom': 14 })
+
+# https://www.openstreetmap.org/node/877270365
+# Rainbow Falls - height 150ft = 45m
+assert_has_feature(
+    14, 4416, 6484, 'pois',
+    { 'kind': 'waterfall', 'min_zoom': 14, 'height': 45.72 })
+
+# https://www.openstreetmap.org/node/404574988
+# Toccoa Falls - height 186' = 56.6929m
+assert_has_feature(
+    13, 2199, 3256, 'pois',
+    { 'kind': 'waterfall', 'min_zoom': 13, 'height': 56.6928 })
```

Comment:
Should we limit heights to ints in the tiles? Or only tenths place? 

In this case it's a foot value that was an int, carrying the original precision thru I'd only expect to see 56 meters.